package.json

```diff
@@ -87,9 +89,12 @@
     "docs:build": "docpress build",
     "commit": "git-cz",
     "cover": "istanbul cover test/index.js",
-    "lint": "eslint bin/ test/ lib/",
+    "lint": "eslint bin/ test/ lib/ .scripts/",
     "test": "npm run lint && node test | tap-spec",
-    "posttest": "rimraf .tmp"
+    "posttest": "rimraf .tmp",
+    "preinstall": "node .scripts/try_rename cached_node_modules node_modules",
```

Comment:
Are you sure about using the `node` binary is a bad thing? Node might be installed in a folder named nodejs on some old systems but it will be accessible via node in the command line nevertheless, won't it?